model-optimizer/extensions/ops/activation_ops.py

```diff
@@ -243,18 +243,15 @@ class Log(Activation):
     operation = staticmethod(lambda x: np.log(x))
 
 
-class SoftPlus(Op):
+class SoftPlus(Activation):
     op = 'SoftPlus'
 
     def __init__(self, graph: Graph, attrs: dict):
-        mandatory_props = {
-            'op': self.op,
-            'type': None,
-            'in_ports_count': 1,
-            'out_ports_count': 1,
-            'infer': None
-        }
-        super().__init__(graph, mandatory_props, attrs)
+        sp_attrs = {'version': 'opset4'}
```

Comment:
nit: Lets make it this simple and handle version setting in the base class
```
class SoftPlus(Activation):
    op = 'SoftPlus'
    version = 'opset4'
    operation = staticmethod(lambda x: np.ln(np.exp(x) + 1.0))
```

Reply:
Take a look how it is done in the https://github.com/openvinotoolkit/openvino/pull/1799 for Reduce ops

Reply:
Done.